### ISSUE TITLE
xccdf_org.ssgproject.content_profile_stig remediations rerun creates duplicates in /etc/audit/rules.d/*.rules

### ISSUE THREAD

**Issue by WrzocHoo** [ComplianceAsCode/content#4844]
#### Description of problem:

**_audit_rules_privileged_commands_** remediation cause duplication of some rules in _/etc/audit/rules.d/privilege.rules_

Reason of that for our case was too open regexp in the following tasks:

```
    - name: Overwrites the rule in rules.d
      lineinfile:
        path: '{{ item.1.path }}'
        line: -a always,exit -F path={{ item.0.item }} -F perm=x -F auid>=1000 -F auid!=unset
          -F key=special-config-changes
        create: false
        regexp: ^.*path={{ item.0.item }} .*$
      with_subelements:
        - '{{ files_result.results }}'
        - files
```

```
    - name: Inserts/replaces the rule in audit.rules
      lineinfile:
        path: /etc/audit/audit.rules
        line: -a always,exit -F path={{ item.item }} -F perm=x -F auid>=1000 -F auid!=unset
          -F key=special-config-changes
        create: true
        regexp: ^.*path={{ item.item }} .*$
      with_items:
        - '{{ files_result.results }}'
      when: ansible_virtualization_role != "guest" or ansible_virtualization_type != "docker"
      tags:
        - audit_rules_privileged_commands
```

Replacing it with the following solved the issue:
```
        regexp: ^.*path={{ item.item }} .* key=special-config-changes$
```

Reason of that is the lineinfile regexp matches ```key=privileged```, which is first and replace it, which cause duplication. Right after ```key=privileged``` rule line is added again, so each time one line per privileged file is added.

#### SCAP Security Guide Version:

Ansible remediation role for profile xccdf_org.ssgproject.content_profile_stig
Profile Title:  DISA STIG for Red Hat Enterprise Linux 7
Profile Description:
This profile contains configuration checks that align to the
DISA STIG for Red Hat Enterprise Linux V1R4.

In addition to being applicable to Red Hat Enterprise Linux 7, DISA recognizes this
configuration baseline as applicable to the operating system tier of
Red Hat technologies that are based off Red Hat Enterprise Linux 7, such as:

- Red Hat Enterprise Linux Server
- Red Hat Enterprise Linux Workstation and Desktop
- Red Hat Enterprise Linux for HPC
- Red Hat Storage
- Red Hat Containers with a Red Hat Enterprise Linux 7 image

Benchmark ID:  xccdf_org.ssgproject.content_benchmark_RHEL-7
Benchmark Version:  0.1.45

XCCDF Version:  1.2

This file was generated by OpenSCAP 1.2.17 using:
	$ oscap xccdf generate fix --profile xccdf_org.ssgproject.content_profile_stig --template urn:xccdf:fix:script:ansible xccdf-file.xml 

#### Operating System Version:

RHEL 7.6

#### Steps to Reproduce:

1. Perform remediation
2. Check /etc/audit/rules.d/privileged.rules
3. Execute: augenrules --load
4. Perform remediation again
5. Check /etc/audit/rules.d/privileged.rules
6. Execute: augenrules --load

#### Actual Results:

Rerun of remediation cause duplicate entries in audit rules, which also triggers error in ```augenrules --load``` command.

#### Expected Results:

There should not be duplicates

#### Addition Information/Debugging Steps:

First run:
```
# grep chage /etc/audit/rules.d/privileged.rules
-a always,exit -F path=/usr/bin/chage -F perm=x -F auid>=1000 -F auid!=unset -F key=privileged
-a always,exit -F path=/usr/bin/chage -F perm=x -F auid>=1000 -F auid!=unset -F key=special-config-changes
```

Second run:
```
# grep chage /etc/audit/rules.d/privileged.rules
-a always,exit -F path=/usr/bin/chage -F perm=x -F auid>=1000 -F auid!=unset -F key=special-config-changes
-a always,exit -F path=/usr/bin/chage -F perm=x -F auid>=1000 -F auid!=unset -F key=special-config-changes
-a always,exit -F path=/usr/bin/chage -F perm=x -F auid>=1000 -F auid!=unset -F key=privileged
```

**Comment by redhatrises** [comment]
@WrzocHoo is this current for the latest version? If you creating issues that are discovered in downstream content that are not fixed in the latest version, we don't support that here.

**Comment by opoplawski** [comment]
I am seeing this with RedHatOfficial.rhel7_stig version 0.1.47 which is the latest in galaxy.

**Comment by opoplawski** [comment]
Though it does appear that this *should* have been fixed with 7a85e0930848344585d5a1f321e69510fd60cf2a in v0.1.45 of this repo.
